compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETile.cpp

```diff
@@ -5,6 +5,7 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 #include "iree-amd-aie/Transforms/Passes.h"
+#include "iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.h"
```

Comment:
Why is this header added?

Reply:
My editor gave a warning about 

```
#include "iree/compiler/Codegen/Common/TileSizeSelection.h"
```

being unused in Passes.h 

Which makes sense: in general you should only include headers in the file that they're needed. This is a well established rule in C++, because it can save unnecessary recompilation when headers change. 

So I removed that header, and added the appropriate headers in the source files they're needed.